search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -91,10 +95,77 @@ function displayTopResultsForCurrentTrend(countryCode) {
     // Get search results for the specified country.
     let results = countryData[0].results;
     for (let i = 0; i < results.length; i++) {
-      resultElement.innerHTML += '<a href=' + results[i].link + '>' +
-        results[i].htmlTitle + '</a><br>' + results[i].snippet + '<br>'
-        + '<i>Sentiment Score: ' + results[i].score.toFixed(1) + '</i><br>';
+      // i + 1 shows the index for each search result. 
+      resultElement.innerHTML += (i + 1).toString() + '. ' + '<a href=' + 
+          results[i].link + '>' + results[i].htmlTitle + '</a><br>' + 
+          results[i].snippet + '<br>' + 'Sentiment Score: ' + results[i].score + 
+          '<br>';
```

Comment:
Get back to the line in master `'<i>Sentiment Score: ' + results[i].score.toFixed(1) + '</i><br>';` where I use italic for the score and truncate the score to have only 1 digit after the decimal point

Reply:
This will be resolved after master is merged into my branch, ill be sure to do so then

Reply:
I am not sure but I think you already merged master into the branch and this is still not fixed.

Reply:
It is fixed?